src/core/core.scale.js

```diff
@@ -47,7 +47,7 @@ module.exports = function(Chart) {
 			labelOffset: 0,
 			// We pass through arrays to be rendered as multiline labels, we convert Others to strings here.
 			callback: function(value) {
-				return helpers.isArray(value) ? value : '' + value;
+				return Chart.Ticks.formatters.values(value);
```

Comment:
I think we can save some bits and a function call: `callback: Chart.Ticks.formatters.values`